spm-monitor/src/main/java/com/sematext/spm/client/json/JsonUtil.java

```diff
@@ -264,13 +270,16 @@ private static Object evaluateFunction(String node, Object element) {
       result = Collections.max(elementList);
     } else if ("min".equals(function)) {
       result = Collections.min(elementList);
+    } else if ("sum".equals(function)) {
+      result = elementList.stream().collect(Collectors.summingDouble(e -> ((Number) e).doubleValue()));
```

Comment:
The supported source level is 1.6 for the agent. lambda expressions are supported from 1.8

Reply:
Eh, right...